wrapper/src/test/java/integration/refactored/TestInstanceInfo.java

```diff
@@ -23,32 +23,36 @@
 public class TestInstanceInfo {
 
   private String instanceId; // "instance-1"
-  private String endpoint; // "instance-1.ABC.cluster-XYZ.us-west-2.rds.amazonaws.com"
-  private int endpointPort;
+  private String host; // "instance-1.ABC.cluster-XYZ.us-west-2.rds.amazonaws.com"
```

Comment:
This change is probably unrelated to leastConnections strategy. I'd make it in a separate PR if needed. We have quite stable integration suite and renaming of properties is definitely can do but it should be a strong reason for doing that.